tests/test_schema_utils.py

```diff
@@ -0,0 +1,37 @@
+from pathlib import Path
+
+import pytest
+from lxml import etree
+
+from tests.schema_utils import (MultipleFilesFound, SchemasDirectoryNotFound,
+                                get_schema_file_path, get_schemas_base_path)
+
+
+def test_get_schemas_base_path_raises_exception() -> None:
+    with pytest.raises(SchemasDirectoryNotFound):
+        get_schemas_base_path('non-existent-schemas-lib')
+
+
+def test_get_schema_file_path_returns_file_path() -> None:
+    """
+    Checks that the schemas are installed, a file path is returned and can
+    be parsed by lxml.
+    """
+    path = get_schema_file_path('item-list.rng')
+    etree.parse(path)
```

Comment:
I like that this doesn't just look at the file, but tries to parse it to see if smoke is coming out